contributing/dataset-ingestion/catalog-ingestion.qmd

```diff
@@ -80,7 +80,7 @@ The following table describes what each of these fields mean:
 </details>
 
 
-> Note: The steps after this are technical, so at this point the scientists can send the json to the VEDAteam and they'll handle the publication process.
+> Note: The steps after this are technical, so at this point the scientists open a PR on the [veda-data](https://github.com/NASA-IMPACT/veda-data) repo and a member of the VEDA team will handle the publication process.
```

Comment:
I just noticed the weird person switch. Maybe:
```suggestion
> Note: The steps after this are technical, so at this point open a PR on the [veda-data](https://github.com/NASA-IMPACT/veda-data) GitHub repository and a member of the VEDA team will handle the publication process.
```

Reply:
Good catch on this one 👍 and both textual comments above. I'll revise and merge. Thanks for your review!